python/publish_transforms.py

```diff
@@ -30,7 +30,12 @@ def publish_transforms(rasgo_api_key: str, rasgo_domain: str) -> None:
 
     # Get all Rasgo Transforms
     rasgo = pyrasgo.connect(rasgo_api_key)
-    rasgo_transforms = utils.get_all_rasgo_transform_keyed_by_name(rasgo)
+
+    org = rasgo.get.organization()
+    if org.name != "Community":
```

Comment:
should we use something like org.id == 1 (and/or make COMMUNITY_ORG_ID = 1 an int in constants.py we can ref)?

Reply:
Reason being: in the future users may be able to change their org name, but we control the id (and 1 is consistent across our envs)